wallet/test/integration/Test/Integration/Scenario/Transactions.hs

```diff
@@ -323,7 +409,7 @@ spec = do
     --   fee+   : 171905         fee+   : 228815
     --
     --           Actual fee: 228903 (+88)
-    coinSelectionScenario "needs many extra inputs" (replicate 8 30000) 42
```

Comment:
Changed because of the `maxTxSize` changed. The test is fundamentally the same, just with less inputs.